src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs

```diff
@@ -93,6 +93,7 @@ private void InitializeNavigationViewItems()
 				AddMenuItem<ButtonSamplePage>(icon: "ButtonIcon");
 				AddMenuItem<CardSamplePage>(icon: "CardsIcon");
 				AddMenuItem<CheckBoxSamplePage>(icon: "CheckboxIcon");
+				AddMenuItem<ChipSamplePage>();
 				AddMenuItem<ComboBoxSamplePage>(icon: "ComboBoxIcon");
```

Comment:
Add icon assets (both divider and placeholder)

Reply:
I cannot find the divider asset. For now, I've only added the placeholder assets